source/lac/lapack_full_matrix.cc

```diff
@@ -696,10 +696,10 @@ LAPACKFullMatrix<number>::vmult(Vector<number> &      w,
                &alpha,
                &this->values[0],
                &mm,
-               v.values.get(),
+               v.data(),
```

Comment:
Was this class a `friend`? Could we unfriend it?

Reply:
Yes.